### PR TITLE
Remic/hargreaves

### PR DESCRIPTION
**Pull request by remicousin**
The rebasing was a bit silly here since I had to reapply all my changes that were exclusively in the delated files...

I didn't put back the tests as I am really not sure what kind of test to run on those two...?

### CODE REVIEW

**Comment by remicousin** [comment]
data_test_calc also got updated by accident. I'll remove those additions if not going to be used.

**Review by aaron-kaplan — COMMENTED** [review]
Looks straightforward. I don't understand the rebasing problems you referred to--provide more details if you want help with that.

Even though the logic of the functions is arguably too straightforward to be worth testing, it's still useful to have one simple test per function to catch typos like misspelled variable names.

File: enacts/wat_bal/agronomy.py [hunk 1]
```diff
@@ -343,4 +343,67 @@ def antecedent_precip_ind(daily_rain, n, time_dim="T"):
...
+    lat : DataArray
+        latitude in radians.
+
+    Returns
+    -------
+    ra : DataArray
+        solar radiation in MJ/m**2/day at latitude `lat` for day of the year `doy` .
+    """
+    distance_relative = 1 + 0.033 * np.cos(2 * np.pi * doy / 365)
+    solar_declination = 0.409 * np.sin(2 * np.pi * doy / 365 - 1.39)
```
[the remark this answers is not in the document]
Comment (remicousin) [reply enacts/wat_bal/agronomy.py:414]: yes, explicitly said in the Allen paper. since it's in cos and sin, there's no discontinuity there.

**Comment by remicousin** [comment]
My comment on rebasing was just a clue that the the commits comments are meaningless...

File: enacts/wat_bal/tests/test_agronomy.py [hunk 2]
```diff
@@ -138,4 +138,31 @@ def test_api_runoff():
...
+
+
+def test_hargreaves_et_ref():
+    et_ref = agronomy.hargreaves_et_ref(
+        xr.DataArray(32),
+        xr.DataArray(5),
+        xr.DataArray(40)
+    )
+
+    assert et_ref == 0.0023 * (32 + 17.8) * np.sqrt(5) * 0.408 * 40
```
Comment (aaron-kaplan) [inline enacts/wat_bal/tests/test_agronomy.py:168]: The red line in a circle indicates that the file doesn't have a newline at the end. This results in spurious diffs when the file is later extended; see line 141 in this diff for an example. You can make future diffs more readable by adding a newline here now.

**Review by aaron-kaplan — COMMENTED** [review]
Looks good other than the irrelevant changes to data_test_calc.py, which I don't understand.

**Comment by remicousin** [comment]
@aaron-kaplan just checking that maybe you thought you had approved already, given your last comment.

**Comment by remicousin** [comment]
ah sorry... they were resolved from the very beginning. I was trying to convey that the individual commits and their comments were useless...

**Pull request merged**